server/lib/ga-collector.js

```diff
@@ -68,7 +68,7 @@ GACollector.prototype = {
       return;
     }
 
-    var self = this;
+    var ownself = this;
```

Comment:
This change is because of https://github.com/divyabiyani/eslint-plugin-fxa/blob/master/lib/configs/client.js#L18